src/root/Root.tsx

```diff
@@ -63,9 +64,11 @@ export const Root = () => {
   return (
     <div className={styles.root}>
       <ErrorBoundary fallbackComponent={GeneralErrorScreen}>
-        <RootMeta />
-        <App />
-        {showPrivacyBanner && <PrivacyBanner closeBanner={closeBanner} />}
+        <ToolsContextContainer>
```

Comment:
Wouldn't it be sufficient to just wrap the `<App />` by `ToolsContextContainer`?